pkg/sql/sem/builtins/replication_builtins.go

```diff
@@ -385,4 +389,84 @@ var replicationBuiltins = map[string]builtinDefinition{
 			Volatility: volatility.Volatile,
 		},
 	),
+	"crdb_internal.split_at": makeBuiltin(
+		tree.FunctionProperties{
+			Category:         builtinconstants.CategorySystemRepair,
+			Undocumented:     true,
+			DistsqlBlocklist: true,
+		},
+		tree.Overload{
+			Types: tree.ParamTypes{
+				{Name: "key", Typ: types.Bytes},
+				{Name: "ttl", Typ: types.Interval},
+			},
+			ReturnType: tree.FixedReturnType(types.Void),
+			Fn: func(ctx context.Context, evalCtx *eval.Context, args tree.Datums) (tree.Datum, error) {
+				if err := evalCtx.SessionAccessor.CheckPrivilege(
+					ctx, syntheticprivilege.GlobalPrivilegeObject, privilege.REPAIRCLUSTER,
+				); err != nil {
+					return nil, err
+				}
+				key := roachpb.Key(tree.MustBeDBytes(args[0]))
+				ttl := tree.MustBeDInterval(args[1])
+				expiration := evalCtx.Txn.DB().Clock().Now().Add(ttl.Nanos(), 0)
+				return tree.DVoidDatum, evalCtx.Txn.DB().AdminSplit(ctx, key, expiration)
+			},
+			Info:       "Splits at an *arbitrary* byte key.",
```

Comment:
Could be good to add a few more words of warning that it is the callers responsibility to make sure the split key is a safe split key.

Reply:
I sorta dislike long descriptions on these; I figured the emphasized "arbitrary" was already saying this?